lib/storage.js

```diff
@@ -53,6 +53,14 @@ async function getAsset(assetReference, directory, name, disableDownload) {
         const source = new Asset(assetReference, directory, name);
         if (disableDownload) {
             console.log(`Skipping source file download for ${AssetComputeLogUtils.redactUrl(source.url)}`);
+            const protocol = URL.parse(assetReference.url).protocol;
+            if (protocol === DATA_PROTOCOL) {
+                await datauri.download(assetReference, source.path);
+                const preSignedUrl = await datauri.getPreSignedUrl(source.path);
+                console.log(`Base64 file uploaded to storage and generated preSignedUrl  using local library : ${preSignedUrl}`);
```

Comment:
We cannot log presigned URLs, as it may leak data.

Reply:
Done